src/main.ts

```diff
@@ -0,0 +1,75 @@
+import * as core from '@actions/core'
+import * as github from '@actions/github'
+
+const oktokit = github.getOctokit(core.getInput('token'))
+
+async function run() {
+  const owner = github.context.repo.owner
+  const repo = github.context.repo.repo
+  const upstream_owner = core.getInput('upstream_owner')
+  const upstream_repo = core.getInput('upstream_repo')
+  const upstream_branch =
+    core.getInput('upstream_branch') ||
+    (await oktokit.rest.repos.get({owner: upstream_owner, repo: upstream_repo}))
+      .data.default_branch
+  const reviewers = core.getInput('reviewers').split(',')
+  const base_branch = (await oktokit.rest.repos.get({owner, repo})).data
+    .default_branch
```

Comment:
suggestion

an idea taken from wei/pull:

our fork could have default_branch set to another branch with custom README (like those readme's in FoG forks that redirects to upstream repo for issues and support) and workflows to avoid conflicts with main branch from upstream. Other files could be even removed completely on that branch.

I'm not sure it is the best idea to solve problem of potential conflicts, just one solution.

For now it is pre-mature care probably, so I'm OK with syncing `upstream_branch` with `default_branch` of fork.

Reply:
Oh that is quite easy to implement so we should probably just implement it when its needed.

Reply:
But if it should be done now, then let me know and I'll do that next or you can add in yourself if you want because I will not be back for a few days. Sorry.